tests/test_tag_matching.py

```diff
@@ -0,0 +1,49 @@
+# Temporary mock setup while waiting for a better way to do integration tests
+from unittest.mock import patch
+
+import pytest
+
+
+def mocked_post_request():
+    return {'cucoo': 'haha'}
+
+
+@patch('requests.post')
+@pytest.fixture(scope='module')
+def tagmatching_result(mock_post):
+    from cognite.tagmatching import tag_matching
+    mock_post.return_value.status_code = 200
+    mock_post.return_value.json.return_value = {'data': {'items': [{'matches': [
+        {'platform': 'a_platform', 'score': 0, 'tagId': 'a_match'},
+        {'platform': 'a_platform', 'score': 0, 'tagId': 'a_match1'},
+        {'platform': 'a_platform', 'score': 0, 'tagId': 'a_match2'}], 'tagId': 'a_tag'}]}}
+    return tag_matching(tag_ids=['a_tag'])
+
+
+def test_object(tagmatching_result):
```

Comment:
How does this work? Doesn't it shadow tagmatching_result ? I tried to run it locally, and it is required... Is this some sort of dependency injection?

Reply:
yes, that is how you inject fixtures into tests.